.github/workflows/gradle.yml

```
@@ -98,6 +98,27 @@ jobs:
+
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Set up JDK ${{ env.JVM_VERSION }}
+        uses: actions/setup-java@v5
+        with:
+          java-version: ${{ env.JVM_VERSION }}
+          distribution: ${{ env.JVM_DISTRIBUTION }}
+
+      - name: 🐘Setup Gradle
+        uses: gradle/actions/setup-gradle@f29f5a9d7b09a7c6b29859002d29d24e1674c884 # v5.0.1
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)